lib/src/package.dart

```diff
@@ -140,6 +140,9 @@ class Package {
   /// This mechanism can be used to avoid loading pubspecs twice. It can also be
   /// used to override a pubspec in memory for trying out an alternative
   /// resolution.
+  ///
+  /// [referringPubspec] is the path to the pubspec that includes this one in
```

Comment:
I don't see this argument passed?

Reply:
Ah yeah, this was from another strategy I tried first. Deleted